db/seeds.rb

```diff
@@ -0,0 +1,18 @@
+Dependency.destroy_all
+Comment.destroy_all
```

Comment:
Whether it's "real" seeds, or fake data we would find useful in a development
workflow, I would not do this, we should never destroy all records for a model
in seeds IMO.